test/integration-tests/IntegrationTests.StartupHook/StartupHookTests.cs

```diff
@@ -0,0 +1,71 @@
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using IntegrationTests.Helpers;
+using IntegrationTests.Helpers.Mocks;
+using IntegrationTests.Helpers.Models;
+using Xunit;
+using Xunit.Abstractions;
+
+namespace IntegrationTests.StartupHook
+{
+    public class StartupHookTests : TestHelper
+    {
+        private List<WebServerSpanExpectation> _expectations = new List<WebServerSpanExpectation>();
+
+        public StartupHookTests(ITestOutputHelper output)
+            : base("StartupHook", output)
+        {
+            SetServiceVersion("1.0.0");
+            _expectations.Add(new WebServerSpanExpectation("Samples.StartupHook", null, "SayHello", "SayHello", null));
+            _expectations.Add(new WebServerSpanExpectation("Samples.StartupHook", null, "HTTP GET", "HTTP GET", null, "GET"));
+        }
+
+        [Theory]
+        [Trait("Category", "EndToEnd")]
+        [InlineData(false)]
+        [InlineData(true)]
+        public void SubmitsTraces(bool enableCallTarget)
+        {
+            SetCallTargetSettings(enableCallTarget);
```

Comment:
I think it is related to IL-level auto-instrumentation which is done via .NET Profiler when JITing. I think it can be removed.

@open-telemetry/dotnet-instrumentation-approvers please confirm if I am correct

Reply:
Based on the enableCallTarget parameter, I think this is intentionally showing that the manual instrumentation is still working regardless of the call target setting. If that's unintentional, or deemed unnecessary that parameter should be removed too.